pdc/apps/package/serializers.py

```diff
@@ -221,3 +222,57 @@ class BuildImageRTTTestsSerializer(StrictSerializerMixin, serializers.ModelSeria
     class Meta:
         model = models.BuildImage
         fields = ('id', 'build_nvr', 'format', 'test_result')
+
+
+class RepositoryField(serializers.SlugRelatedField):
+
+    def __init__(self, **kwargs):
+        super(RepositoryField, self).__init__(slug_field='id', queryset=Repo.objects.all(), **kwargs)
+
+    def to_representation(self, value):
+        export_data = value.export()
+        export_data["id"] = value.id
+        export_data["content_format"] = {"name": value.content_format.name,
+                                         "pdc_endpoint": value.content_format.pdc_endpoint}
```

Comment:
This field should be visible in top level of the new endpoint instead.

Reply:
Which level you mean?
Currently there is a content_format in repo, so we don't need add a content format again in release-files, just get the repo means get the content_format, and also if we move the pdc_endpoint out in repo, this is also not good because if you have looked at the repo's api you can see there is just content_format.name here, so I think keep it here is more clearly, and that's why I put a dict in the docstring.

Reply:
# This will make APIs inconsistent. Maybe we can improve RepoField to take simple ID value in addition to dict.
For me, I think we'd better just keep it here, sync the two classes will break other api. 
We can create an new issue to do this.

Reply:
This changes the way `content_format` is serialized - it can break some client code which expects the type to be simple string. Also, I think it's more difficult for user to access the `pdc_endpoint` value. So I propose to serialize it with released-files serializer directly and have it on the top level as `repo` field.

Reply:
That makes sense, I have moved it to the top.